src/commands/update.ts

```diff
@@ -279,7 +322,7 @@ export default class UpdateCommand extends Command {
       .on('error', reject)
       .on('close', (status: number) => {
         try {
-          this.exit(status)
+          if (status > 0) this.exit(status)
```

Comment:
does this mean `status=0` won't exit?

Reply:
Ya it will exit normally. Why does exit even need to be called here? I did this because it was printing an EXIT error to the terminal even though it was status 0. Why does exit return an error when it is 0? https://github.com/oclif/command/blob/master/src/command.ts#L119